{
  return Closure.this.throwRuntimeException(throwable);
}
